src/components/MovieDetails/index.tsx

```diff
@@ -111,6 +117,9 @@ const MovieDetails: React.FC<MovieDetailsProps> = ({ movie }) => {
     return <Error statusCode={404} />;
   }
 
+  const studios = data.productionCompanies;
```

Comment:
I don't think this is necessary 😅

Can also revert the changes on lines 631, 635, and 639.

Reply:
👍